tests/testing_utils_test.go

```diff
@@ -17,8 +17,10 @@ func runWithTimeout(
 		timeoutEvent := time.After(timeout)
 		done := make(chan bool)
 		go func() {
+			defer func() {
```

Comment:
Apparently run(t) panics occasionally making tests hang; so defer is the right thing to do.